apps/web/content/docs/en/core/fees.mdx

```diff
@@ -5,117 +5,107 @@ description:
 ---
-
-Every Solana transaction requires a base fee (SOL) to compensate validators for
+## Overview
+Every Solana transaction requires a base fee (paid in SOL) to compensate validators for
 processing the transaction. You can also pay an optional prioritization fee to
 increase the chance that the current leader (validator) processes your
 transaction.
 
-## Key Points
-
-- The **base fee** for a transaction is 5000 lamports per signature on the
-  transaction.
-- The **prioritization fee** (optional) is an extra fee you pay to the validator
-  to increase the chance that the current leader processes your transaction.
-- The prioritization fee equals: **(compute unit limit × compute unit price)**.
-- The **compute unit limit** is the maximum compute units your transaction can
-  use.
-- The **compute unit price** is the price per compute unit, in micro-lamports.
-- 1,000,000 micro-lamports = 1 lamport
-- The transaction fee payer must be an account owned by the System Program.
-
-## Base Transaction Fee
+## Base transaction fee
 
 The base fee is the cost to send a transaction. The cost is 5000 lamports per
 signature included in the transaction.
 
-The base fee comes out of the transaction fee payer's account, which is the
-first signer on the transaction. The fee payer must be an account owned by the
-System Program.
+The base fee is paid by the first signer on the transaction. The fee payer must be an account
+owned by the System Program.
+
+Solana splits each base fee as follows: 
+
+- **50% burned:** Half of the base fee gets taken out of the circulating SOL supply.
+- **50% distributed:** Half is paid to the validator that processed the transaction.
 
-- [50% Burned](https://github.com/anza-xyz/agave/blob/v2.1.13/sdk/fee-calculator/src/lib.rs#L70):
-  Half of the base fee gets burned.
-- [50% Distribution](https://github.com/anza-xyz/agave/blob/e621336acad4f5d6e5b860eaa1b074b01c99253c/runtime/src/bank/fee_distribution.rs#L58-L62):
-  Half is paid to the validator that processed the transaction.
+See relevant code for [burning 50%](https://github.com/anza-xyz/agave/blob/v2.1.13/sdk/fee-calculator/src/lib.rs#L70) 
+and [distributing 50%](https://github.com/anza-xyz/agave/blob/e621336acad4f5d6e5b860eaa1b074b01c99253c/runtime/src/bank/fee_distribution.rs#L58-L62) 
+of the base fee.
 
-## Prioritization Fee
+## Prioritization fee
 
 The prioritization fee is an optional fee paid to increase the chance that the
-current leader processes your transaction.
+current leader processes your transaction, which will increase the speed of your transaction. 
+The validator processing the transaction receives 100% of the priority fee.
 
-- [SIMD-0096](https://github.com/solana-foundation/solana-improvement-documents/blob/main/proposals/0096-reward-collected-priority-fee-in-entirety.md):
-  The validator processing the transaction receives 100% of the priority fee.
+{/* I removed the link, because it just shows the 2023 proposal to change the allocation of 
```

Comment:
Let's keep the link. It looks like SIMD stands for Solana Improvement Documents so I do think they probably want to encourage people to interact with those.

can we change the link text to be on the [priority fee](https://github.com/solana-foundation/solana-improvement-documents/blob/main/proposals/0096-reward-collected-priority-fee-in-entirety.md) though?